src/main/java/frc/robot/subsystems/PivotSubsystem.java

```diff
@@ -1,144 +1,137 @@
 package frc.robot.subsystems;
 
-import com.revrobotics.CANSparkBase;
-import com.revrobotics.CANSparkLowLevel;
-import com.revrobotics.RelativeEncoder;
-import edu.wpi.first.math.controller.ArmFeedforward;
-import edu.wpi.first.math.controller.ProfiledPIDController;
+import com.revrobotics.*;
 import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
-
-import com.revrobotics.CANSparkFlex;
 import edu.wpi.first.wpilibj2.command.Command;
-import edu.wpi.first.wpilibj2.command.InstantCommand;
-import edu.wpi.first.wpilibj2.command.ProfiledPIDSubsystem;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
 import frc.robot.Constants.Constants.PivotConstants;
 
 
+import static frc.robot.Constants.Constants.PivotConstants.pivotEncoderChannel;
+
+public class PivotSubsystem extends SubsystemBase {
+
+    //makes motors
+    public final CANSparkFlex pivotMotor1 = new CANSparkFlex(PivotConstants.pivotMotor1ID, CANSparkLowLevel.MotorType.kBrushless);
+    public final CANSparkFlex pivotMotor2 = new CANSparkFlex(PivotConstants.pivotMotor2ID, CANSparkLowLevel.MotorType.kBrushless);
+
+    //gets encoders
+    public final RelativeEncoder pivotEncoder = pivotMotor1.getEncoder();//change later
 
```

Comment:
I don't think we're still changing this, remove

Reply:
wait what do you want to remove this is the encoder.